components/concept/engine/src/main/java/mozilla/components/concept/engine/webextension/WebExtensionsTabsDelegate.kt

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.concept.engine.webextension
+
+import mozilla.components.concept.engine.EngineSession
+
+/**
+ * Provides delegation for when a extension opens or closes a tab.
+ */
+interface WebExtensionsTabsDelegate {
```

Comment:
nit: The double plural is a bit hard to read. Let's refactor to just `WebExtensionTabDelegate` similar to GeckoView.